src/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { history } from '@onaio/connected-reducer-registry';
+import { ConnectedRouter } from 'connected-react-router';
+import { Provider } from 'react-redux';
+import './index.css';
+import App from './App/App';
+import * as serviceWorker from './serviceWorker';
+import store from './store';
+
+// tslint:disable-next-line: ordered-imports
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Do we need to import this if we are now using Ant?

Reply:
This is temporarily needed since we are using the js tools reactstrap table for listing users, we shall decide on this once we discuss https://github.com/OpenSRP/web/issues/26